reoptjl/test/posts/central_plant_ghp.json

```diff
@@ -53,7 +53,12 @@
             "tess_ghx_minimum_timesteps_per_hour": 1,
             "max_sizing_iterations": 10,
             "init_sizing_factor_ft_per_peak_ton": 300.0,
-            "heat_pump_configuration": "WWHP"   
+            "heat_pump_configuration": "WWHP",
```

Comment:
what does this ghp stuff have to do with this PR?

Reply:
A month or so ago, An updated several defaults for GHP in [NREL/GhpGhx.jl](https://github.com/NREL/GhpGhx.jl/commit/fb3d8f5321ee8476952d81f1ef4d5d96ea20dc5e) and also the ghpghx/models.py file here. However, tests using this post continued to pass because the GhpGhx pkg in REopt.jl had not been updated until I recently ran "update" in the feature branch this is pointing to. So, the tests using this post were failing on this branch until I fixed the defaults to the old values